merchants/verkkokauppa/tests/test_product_requests.py

```diff
@@ -1,60 +1,144 @@
 from contextlib import suppress
+from typing import Dict
 from unittest.mock import Mock
+from uuid import UUID
 
 from assertpy import assert_that
 from django.conf import settings
+from django.test.testcases import TestCase
 from pytest import raises
 from requests import Timeout
 
 from ..constants import REQUEST_TIMEOUT_SECONDS
-from ..product.exceptions import CreateProductError
-from ..product.requests import create_product
-from ..product.types import Product
-from ..tests.mocks import mock_post
+from ..product.exceptions import CreateProductError, GetProductMappingError
+from ..product.requests import create_product, get_product_mapping
+from ..product.types import CreateProductParams, Product
+from ..tests.mocks import mock_get, mock_post
 
 
-def test_create_product_makes_valid_request(create_product_params):
-    post = mock_post({})
-    with suppress(Exception):
-        create_product(create_product_params, post)
-    post.assert_called_with(
-        url=settings.VERKKOKAUPPA_PRODUCT_API_URL,
-        json=create_product_params.to_json(),
-        headers={"api-key": settings.VERKKOKAUPPA_API_KEY},
-        timeout=REQUEST_TIMEOUT_SECONDS,
-    )
+class ProductRequestsBaseTestCase(TestCase):
```

Comment:
totally nitpicking :nerd_face: => would consider renaming to `ProductRequestsTestCaseBase` as its the base and not TestCase

Reply:
Nitpick approved and class renamed 😄